src/cli/commands/sendMessage.ts

```diff
@@ -39,16 +39,12 @@ export class SendMessage extends Command<ServerContext> {
 
     const now = new Date()
     const sender = makePushSender(connection)
-    const status = await sender.send(
+    await sender.send(
       connection,
       { title, body: this.body },
       { date: now, deviceId, loginId }
     )
 
-    stdout.write(
-      `Sent to ${status.successCount} devices and failed to reach ${status.failureCount} devices.\n`
-    )
```

Comment:
Using our standard heartbeat utility for progress feedback. Since we are moving to a queue, we don't actually have any stats about a specific device, only the overall progress of the loop.